packages/Sortable/stories/examples/Basic.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Story } from "storybook/assets/styles/common.styles";
+import { storyStyles } from "../Sortable.stories.helpers";
+import Sortable from "../../src";
+
+const Example = () => {
+  return (
+    <Story css={storyStyles}>
+      <Sortable onChange={() => {}} onRemove={() => {}}>
+        <div>Item One</div>
```

Comment:
I wonder if you'd ever want to have some children that are not sortable...

Reply:
To me sound like if we have a <Sortable.Item> subcomponent will be clearer to the consumer what to do:

if the component accepts children then a consumer might do:
```js
<Sortable>
 { ["one", "two", "three"].map(i => i) }
</Sortable>
```
___ or ___

```js
<Sortable>
 { ["one", "two", "three"].map(i => <span>i</span>) }
</Sortable>
```

both cases are inline elements don't know if the drag and drop will work on those situations without being wrapped in a div.

To me is more idiomatic to have:

```js
<Sortable>
 { ["one", "two", "three"].map(i => <Sortable.Item>{i}</Sortable.Item>) }
</Sortable>
```

^ where we can control exactly the tag to render and the consumer don't have to worry about what tag to render. 

my `giveAFu` thermometer ★★★★ 4/5

Reply:
We are controlling the elements already though because `Item` wraps the children

Reply:
An array of Inline `<spans>` or strings, even `null`, `undefined`, or `[]` are handled.  Strings are handled by wrapping them in `<spans>` in `augmentChildren()`:

https://github.com/acl-services/paprika/blob/0e07cf3d93d346d3737d96efe0f36092880def91/packages/Sortable/src/Sortable.js#L24-L26

So there aren't any issues. Imposing this restriction is not technically necessary, and the implementation will appear redundant. The `giveAFu` rating is high though, so maybe others can weigh in, and/or we can discuss in person.

Reply:
@jamiek-acl, there's a way to disable individual items with this library.  Depending on the use case, we could implement it in a visual (greyed out), or subtle way (it just doesn't move).  But currently, we don't have this use case, so I'm not implementing it at this time (but I did experiment a little to make sure it would be easy to add later).

Reply:
just playing `devil advocate`:

how would you mark `isDisabled` in a specific child with this API while using divs and spans?

// to me this is cleaner:
<Sortable>
  <Sortable.Item isDisable><MyComponent /></Sortable.item>
</Sortable>

// current api
// seems odd to pass isDisabled over a div/span/p. 
<Sortable>
   <div isDisabled><MyComponent /></div>
   <div isDisabled><MyComponent /></div>
</Sortable>

Reply:
Exactly @nahumzs , that's what I was getting at. I think it is more clear for the children to be `<Sortable.Item>`s.

Reply:
Okay, that makes some sense. It also means I can rename `data-drag-id` → `dragId`.